### DISCUSSION TITLE
Override sticky 1st column header on a table.

### DISCUSSION THREAD

**Discussion by MisterB80** [saadeghi/daisyui#1601] [Q&A] [answered]
Hi, I'm using the table and have noticed that the first column header is made sticky on horizontal scroll. Is there any way to override this? I have a particularly long string in my first column and it means that none of the other column headers are visible when I scroll them into view.

This is the css.

`.table th:first-child {
    position: sticky;
    left: 0px;
    z-index: 11;
    /* because safari */
    position: -webkit-sticky
}`

**Comment by MarkoRakic88** [comment]
I have same problem. Please help.

**Comment by MisterB80** [comment]
Has anyone got a solution for this yet? It's becoming more of a problem for me now.

**Comment by MisterB80** [comment, marked as answer]
Ok, I've solved this by adding the following CSS to my main.css after the @tailwind directives.

```
.table th:first-child {
    position: static;
}
```
